omod/src/main/java/org/openmrs/module/rwandahivflowsheet/web/UIHelper.java

```diff
@@ -84,10 +84,10 @@ public static String formatRegimenDisplaySummary(DrugOrder drugOrder) {
 		Double dose = drugOrder.getDose();
 		String units = "";
 		String frequency = "";
```

Comment:
This is the best you can do for now, but this might not actually do what it is meant to do following the migration to coded units and frequencies, since this is pretty hacky code written specifically for names of free-text data.